packages/babel-preset-cgb/index.js

```diff
@@ -30,35 +29,38 @@ module.exports = {
 		],
 	],
 	plugins: [
-		// class { handleClick = () => { } }.
-		[ require.resolve( 'babel-plugin-transform-class-properties' ) ],
-		// The following two plugins use Object.assign directly, instead of Babel's
-		// extends helper. Note that this assumes `Object.assign` is available.
-		// { ...todo, completed: true }
+		'@babel/plugin-syntax-dynamic-import',
+		'@babel/plugin-syntax-import-meta',
 		[
-			require.resolve( 'babel-plugin-transform-object-rest-spread' ),
+			'@babel/plugin-proposal-object-rest-spread',
 			{
 				useBuiltIns: true,
 			},
 		],
 		[
-			// Transforms JSX Syntax.
-			// 'transform-react-jsx',
-			require.resolve( 'babel-plugin-transform-react-jsx' ),
+			'@babel/plugin-proposal-class-properties',
 			{
-				pragma: 'wp.element.createElement',
+				loose: true,
 			},
 		],
-		// Async/Await awesomeness https://babeljs.io/docs/en/babel-plugin-syntax-async-functions/.
-		[ require.resolve( 'babel-plugin-syntax-async-functions' ) ],
```

Comment:
Why removing this?

Reply:
In looking into this one, I discovered that there is no @babel equivalent of this plugin. Moreover, I believe the functionality desired is present, unless we need more than just supporting async functions. Do we need to include @babel/plugin-transform-async-to-generator ? 

------------ reference -----------------
https://github.com/babel/babel/issues/9017

"That package has been deprecated and is not maintained anymore, since parsing async functions is now enabled by default. If you want to transform them, you should use @babel/plugin-transform-async-to-generator."

Reply:
since parsing async functions is now enabled by default we can just let go of it.